README.md

```diff
@@ -88,38 +101,41 @@ To create service credentials, access to Event Notifications and access to call
+| <a name="input_region"></a> [region](#input\_region) | The IBM Cloud region where the Event Notifications resource is created. The following regions are supported: `us-south` (Dallas), `eu-gb` (London), `eu-de` (Frankfurt), `au-syd` (Sydney), `eu-es` (Madrid) | `string` | `"us-south"` | no |
+| <a name="input_resource_group_id"></a> [resource\_group\_id](#input\_resource\_group\_id) | The ID of the resource group where the Event Notifications instance is created. | `string` | n/a | yes |
+| <a name="input_root_key_id"></a> [root\_key\_id](#input\_root\_key\_id) | The key ID of a root key, existing in the key management service (KMS) instance that is passed in `var.existing_kms_instance_crn`, which is used to encrypt the data encryption keys (DEKs) which are then used to encrypt the data. This variable is required if `var.kms_encryption_enabled` is set to `true`. | `string` | `null` | no |
+| <a name="input_service_credential_names"></a> [service\_credential\_names](#input\_service\_credential\_names) | The mapping of names and roles for service credentials that you want to create for the Event Notifications instance. | `map(string)` | `{}` | no |
+| <a name="input_service_endpoints"></a> [service\_endpoints](#input\_service\_endpoints) | Specify whether you want to enable public, or both public and private service endpoints. The following values are supported: `public`, `public-and-private` | `string` | `"public-and-private"` | no |
+| <a name="input_skip_iam_authorization_policy"></a> [skip\_iam\_authorization\_policy](#input\_skip\_iam\_authorization\_policy) | Set this input value to `true` to skip the creation of an IAM authorization policy that permits all Event Notifications instances in the resource group to read the encryption key from the key management service (KMS) instance. No policy is created if `var.kms_encryption_enabled` variable is set to `false`. | `bool` | `false` | no |
 | <a name="input_tags"></a> [tags](#input\_tags) | Optional list of tags to be added to the Event Notification instance | `list(string)` | `[]` | no |
 
 ### Outputs
+{: #EN-outputs}
 
 | Name | Description |
 |------|-------------|
-| <a name="output_crn"></a> [crn](#output\_crn) | Event Notification crn |
-| <a name="output_event_notification_instance_name"></a> [event\_notification\_instance\_name](#output\_event\_notification\_instance\_name) | Event Notification name |
-| <a name="output_guid"></a> [guid](#output\_guid) | Event Notification guid |
-| <a name="output_service_credentials_json"></a> [service\_credentials\_json](#output\_service\_credentials\_json) | Service credentials json map |
-| <a name="output_service_credentials_object"></a> [service\_credentials\_object](#output\_service\_credentials\_object) | Service credentials object |
+| <a name="output_crn"></a> [crn](#output\_crn) | The Cloud Resource Name (CRN) of the Event Notifications instance. |
```

Comment:
In general, I don't spell out abbreviations for things like CRNs just because of the space limitations in the projects UI. I do make sure they are capitalized, tho.